### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by ratzel921**

### CODE REVIEW

**Comment by immae1** [comment]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -9,11 +10,13 @@ RUN apt-get update && \
...
 
 # Config SSH-Server
-RUN mkdir /var/run/sshd && \
-    echo 'root:root' | chpasswd && \
-    sed -ri 's/^#?PermitRootLogin\s+.*/PermitRootLogin yes/' \
-    /etc/ssh/sshd_config && \
-    sed -ri 's/UsePAM yes/#UsePAM yes/g' /etc/ssh/sshd_config \
+RUN mkdir /var/run/sshd \
+    && useradd --create-home --shell /bin/bash novatec && echo "novatec:Schulung2022" | chpasswd && adduser novatec sudo \
+    && echo 'root:root' | chpasswd \
```
Comment (jakobyte1024) [inline Dockerfile:21]: can you provide some comments in the Dockerfile for "why is this is necessary", please

Comment (ratzel921) [reply Dockerfile:21]: done

Comment (immae1) [reply Dockerfile:21]: provide password pls as "secret" not in cleartext...

**Pull request merged**